react-design/react-design-app/src/RotatingBanner.jsx

```diff
@@ -0,0 +1,41 @@
+export default function RotatingBanner({ items }) {
+  function Banner() {
+    return <p>{items}</p>;
+  }
+
+  function PrevButton() {
+    return <button>Prev</button>;
+  }
+
+  function NextButton() {
+    return <button>Next</button>;
+  }
+
+  function IndicatorButton() {
+    return (
+      <>
+        <button>0</button>
```

Comment:
No need to change for Phase 1, this hard coding for now works as per the assignment instructions

But what if we needed to hardcode 50 buttons, about about 100? Ask yourself this and you'll code this out more in Phase 2.